### PR TITLE
Added the url for uu.nl domains

### PR DESCRIPTION
**Pull request by SanderVerkuil**
Added the git clone URL for the University of Utrecht. The basic url is:

```
<studentnumber>@students.science.uu.nl:/home/projects/<ProjectTitle>/git/<GitRepo>.git
```

The studentnumber is your student number, the ProjectTitle is the title assigned to you by the system administrators of the university of Utrecht and the git/<GitRepo>.git is the file you created during the `$ git init` stage. This should be done via SSH in the projects location. Only you and your team members have access to this file. You will be asked for a password, which is quite logical.

Basically, the git folder and the .git aren't needed per se, but it is just used for managing your projects space to be clean and nice.

### CODE REVIEW

**Comment by wouter-swierstra** [comment]
Thanks Sander!

**Comment by DouweM** [comment]
That's really cool, I had no idea the project spaces supported SSH.
